### PR TITLE
feat: add wallet chain select

### PR DESCRIPTION
**Pull request by 6heLL66**

### CODE REVIEW

File: src/widgets/modals/selectWallet/ui/components/ChainSelect/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  const handleChange = (e: SelectChangeEvent<ChainType>): void => {
+    setValue(e.target.value as ChainType)
+    onChainChanged?.(e.target.value as ChainType)
+  }
+
+  return (
+    <Select<ChainType> value={value} onChange={handleChange}>
+      {options.map(option => {
+        return <SelectItem value={option.value}>{option.label}</SelectItem>
```
Comment (ordinaryask) [inline src/widgets/modals/selectWallet/ui/components/ChainSelect/index.tsx:24]: without return?

**Review by ordinaryask — CHANGES_REQUESTED** [review]
add new phantom icon and SUI wallet icon or suiet idk

File: src/widgets/modals/selectWallet/ui/components/ChainSelect/index.tsx (hunk 1 above)
Comment (6heLL66) [reply src/widgets/modals/selectWallet/ui/components/ChainSelect/index.tsx:24]: no :)

File: src/shared/assets/icons/solana_icon.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { IconProps } from 'shared/assets/types'
+
+export const SolanaIcon = ({ ...props }: IconProps): JSX.Element => {
```
Comment (ordinaryask) [inline src/shared/assets/icons/solana_icon.tsx:3]: export from index

File: src/shared/assets/icons/select_arrow.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { IconProps } from 'shared/assets/types'
+
+export const SelectArrow = ({ ...props }: IconProps): JSX.Element => {
```
Comment (ordinaryask) [inline src/shared/assets/icons/select_arrow.tsx:3]: export from index

File: src/shared/assets/icons/eth_icon.tsx [hunk 4]
```diff
@@ -0,0 +1,33 @@
+import { IconProps } from 'shared/assets/types'
+
+export const EtheriumIcon = ({ ...props }: IconProps): JSX.Element => {
```
Comment (ordinaryask) [inline src/shared/assets/icons/eth_icon.tsx:3]: export from index

**Review by ordinaryask — CHANGES_REQUESTED** [review]

**Review by ordinaryask — APPROVED** [review]

**Pull request merged**